libraries/botbuilder-dialogs-adaptive/src/adaptiveComponentRegistration.ts

```diff
@@ -338,9 +338,9 @@ export class AdaptiveComponentRegistration extends ComponentRegistration impleme
     }
 
     /**
-     * @private
+     * @protected
      */
-    private _addDeclarativeType<T, C>(type: Type<T>, loader?: CustomDeserializer<T, C>): void {
+    protected _addDeclarativeType<T, C>(type: Type<T>, loader?: CustomDeserializer<T, C>): void {
```

Comment:
Same feedback as https://github.com/microsoft/botbuilder-js/pull/3150/files#r543636380

Reply:
Fixing. In trying to get this to work w/ Composer, I played around with different types of inheritance. This is a leftover, unneeded correction. Turns out that wasn't the issue, anyway. Will fix.